### ISSUE TITLE
[NFR]Configuration file

### ISSUE THREAD

**Issue by Jurigag** [phalcon/phalcon-devtools#980]
I would like to see some option to have configuration file for phalcon devtools which could be read. Maybe something like
`devtools.json` or `phalcon.json` or something like this in project root where we can configure all options for all commands so for example:
```
{
    migration: {
        config: 'App/Config/db.ini',
        migrations: 'App/Migrations'
    }
}
```

Running just `phalcon migration generate` will have same effect as
`phalcon migration generate --config=App/Config/db.ini --migrations=App/Migrations`

Or maybe there is already a way to do this?

**Comment by Jurigag** [comment]
It's not only about migrations, this was just example, i would like to see setting it for any command.

**Comment by Jurigag** [comment]
Well created PR with it.

**Comment by Jurigag** [comment]
Implemented in 3.1.x
